tests/testthat/test-tidyme.R

```diff
@@ -75,8 +75,9 @@ testthat::test_that("T3.2 The S3 method, associated with a `survfit` object, has
 
   survfit_object <- visR::estimate_KM(data = adtte, strata = "TRTA")
   survfit_object_tidy <- tidyme(survfit_object)
+  survfit_object$strata_lbls <- NULL
```

Comment:
This is cheating :) You need to add the labels to surv_object_df to demonstrate that each list element is available in the final dataframe

Reply:
corrected